### ISSUE TITLE
Crash on completing to-do

### ISSUE THREAD

**Issue by oorzkws** [LuziferSenpai/Not-Enough-Todo#1]
I checked off the top item in the todo-list shown in the below screenshot, and received an error.
[factorio-current.log](https://github.com/LuziferSenpai/Not-Enough-Todo/files/5146414/factorio-current.log)
![image](https://user-images.githubusercontent.com/65210810/91652870-42629500-ea58-11ea-8a71-ad94a01ab954.png)
[_autosave1.zip](https://github.com/LuziferSenpai/Not-Enough-Todo/files/5146412/_autosave1.zip)

**Comment by dcbishop** [comment]
Same here also

```
The mod Not Enough Todo - NET (0.0.2) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event Not_Enough_Todo::on_gui_checked_state_changed (ID 3)
LuaGuiElement API call when LuaGuiElement was invalid.
stack traceback:
	__Not_Enough_Todo__/scripts/events.lua:122: in function 'handler'
	__core__/lualib/event_handler.lua:47: in function <__core__/lualib/event_handler.lua:45>
stack traceback:
	[C]: in function '__newindex'
	__Not_Enough_Todo__/scripts/events.lua:122: in function 'handler'
	__core__/lualib/event_handler.lua:47: in function <__core__/lualib/event_handler.lua:45>
```

**Comment by LuziferSenpai** [comment]
Will be fixed in 1.0, when that comes out.
